treeherder/perf/models.py

```diff
@@ -129,25 +202,23 @@ class PerformanceAlert(models.Model):
     id = models.AutoField(primary_key=True)
     summary = models.ForeignKey(PerformanceAlertSummary,
                                 related_name='alerts')
-    revised_summary = models.ForeignKey(PerformanceAlertSummary,
-                                        related_name='revised_alerts',
+    related_summary = models.ForeignKey(PerformanceAlertSummary,
+                                        related_name='related_alerts',
                                         null=True)
     series_signature = models.ForeignKey(PerformanceSignature)
     is_regression = models.BooleanField()
 
     UNTRIAGED = 0
-    INVALID = 1
-    WONTFIX = 2
-    INVESTIGATING = 3
-    RESOLVED = 4
-    DUPLICATE = 5
+    DOWNSTREAM = 1
+    REASSIGNED = 2
+    INVALID = 3
+    ACKNOWLEDGED = 4
 
     STATUSES = ((UNTRIAGED, 'Untriaged'),
+                (DOWNSTREAM, 'Downstream'),
+                (REASSIGNED, 'Reassigned'),
                 (INVALID, 'Invalid'),
-                (WONTFIX, 'Won\'t fix'),
-                (RESOLVED, 'Resolved'),
-                (INVESTIGATING, 'Investigating'),
-                (DUPLICATE, 'Duplicate'))
+                (ACKNOWLEDGED, 'Acknowledged'))
```

Comment:
why are these different from the statuses above?  do we have different status for summary vs alert?